django/contrib/contenttypes/models.py

```diff
@@ -174,20 +174,23 @@ def model_class(self):
         except LookupError:
             return None
 
-    def get_object_for_this_type(self, **kwargs):
+    def get_object_for_this_type(self, using=None, **kwargs):
```

Comment:
This method is [documented](https://docs.djangoproject.com/en/5.0/ref/contrib/contenttypes/#django.contrib.contenttypes.models.ContentType.get_object_for_this_type), so any change in the signature should be reflected in docs.

Reply:
@felixxm I updated the docs.